core/unit_test/tstLinkedCellList.hpp

```diff
@@ -237,6 +236,437 @@ void testLinkedList()
     }
 }
 
+//---------------------------------------------------------------------------//
+void testLinkedListRange()
+{
+    // Make an AoSoA with positions and ijk cell ids.
```

Comment:
For the construction part of the test it looks like this is reused from the previous test - maybe we can break this into a separate function for code reuse.

Reply:
Agreed. I'll make one function to set up the problem, then reuse throughtout